test/paper-button-behavior.html

```diff
@@ -95,13 +69,12 @@
             }
           });
         });
-
-        test('receives focused', function() {
-          MockInteractions.focus(button);
-          assert.equal(button._elevation, 2);
-        })
       });
 
+      test('receives focused', function() {
+        MockInteractions.focus(button);
+        assert.equal(button._elevation, 3);
```

Comment:
Keyboard focus is the same as "focus from something that isn't a pointer." So it does work correctly.